docs/src/pages/[platform]/connected-components/authenticator/quick-start.vue.mdx

```diff
@@ -0,0 +1,35 @@
+import { Fragment } from '@/components/Fragment';
+
+### Step 1. Configure backend
+
+<Fragment useCommonWebContent>
+  {({ platform }) => import(`./quick-start-pull.${platform}.mdx`)}
+</Fragment>
+
+### Step 2. Install dependencies
+
+<Fragment useCommonWebContent>
+  {({ platform }) => import(`./quick-start-install.${platform}.mdx`)}
+</Fragment>
+
+### Step 3. Update Vite config and index.html
+
+When working with Vite in a Vue project, you must make a few modifications. Please follow the steps below.
+
+<Fragment>
+  {({ platform }) => import('../../getting-started/troubleshooting/shared/vite-polyfill.mdx')}
+</Fragment>
+
+<Fragment>
+  {({ platform }) => import('../../getting-started/troubleshooting/shared/vite-polyfill-ts.mdx')}
```

Comment:
I think this piece needs to show conditionally inside of a TypeScript Tab?

Reply:
Oh or do you mean it should be like on the React troubleshooting steps with Vite? https://ui.docs.amplify.aws/react/getting-started/troubleshooting#vite

Reply:
This is fixed in latest commits and updated here: https://addpolyfillsteps.dvmvffzts1tcu.amplifyapp.com/vue/connected-components/authenticator#step-3-update-vite-config-and-indexhtml